addons/mail/static/src/core/common/attachment_view.js

```diff
@@ -73,13 +71,25 @@ export class AttachmentView extends Component {
     }
 
     popoutAttachment() {
-        this.mailPopoutService.popout(this.__owl__.bdom.parentEl).focus();
+        class PopupWrapper extends Component {
+            static template = xml`<div class="o_attachment_preview" style="display: block;"><AttachmentView t-props="props"/> </div>`;
+            static components = {
+                AttachmentView,
+            };
+        }
+
+        const app = new App(PopupWrapper, {
+            name: "Popup",
+            env: this.env,
+            props: this.props,
+            getTemplate,
+        });
+        this.mailPopoutService.popout(app, this.toggleVisibility.bind(this));
     }
 
-    updatePopup() {
-        if (this.mailPopoutService.externalWindow) {
-            this.mailPopoutService.popout(this.__owl__.bdom.parentEl, false);
-        }
+    toggleVisibility() {
+        const oldDisplay = this.__owl__.bdom.parentEl.style.display;
```

Comment:
this.__owl__ is an internal of owl and it shouldn't be use.
you can, if you need use t-ref.